io/disk/disktest.py

```diff
@@ -122,7 +126,7 @@ def _init_params(self):
         dmesg.clear_dmesg()
         self.pre_cleanup()
         if self.disk is not None:
-            if self.disk in disk.get_disks():
+            if self.disk in self.sysdisks:
```

Comment:
just small suggestions,
it would have been good if we could check for exact match here. this will be helpful when we have partition on the disks like below,

/dev/disk/by-id/scsi-360050768108001b3a80000000000010b-part3'
if we pass `scsi-360050768108001b3a80000000000010b` it will match the multiple instance here.
and while trying access it with actual name, then it may not work.

Reply:
@Naresh-ibm is there any problem to use block device instead of its already created partitions ?

Reply:
@Naresh-ibm I tested, basically when you are matching a string to list using 'in' than it is exact match .. so we are good here